lua/websitebypass/websitebypass_config.json

```diff
@@ -1,5 +1,7 @@
 {
-  "use_webdriver": false,
-  "testing": false,
-  "debug": false
+	"use_webdriver": false,
+	"debug": false,
+	"flaresolverr_port": 8191,
+	"testing": false,
+	"flaresolverr_ip": "localhost"
```

Comment:
Looking at `config_table`, the output should be like this:

```suggestion
	"use_webdriver": false,
	"testing": false,
	"debug": false,
	"flaresolverr_ip": localhost,
	"flaresolverr_port": "8191"
```

Reply:
are you talking about '"flaresolverr_port": "8191"'?
the code can work in both int and str port number, so it doesn't not mater if it "8191" or 8191

Reply:
ohh, but we are using json object and According to the official JSON specification (RFC 7159), a JSON object is an unordered collection of name/value pairs
that why it depend of json.lua we use, even if we order it when we use json.encode it will put it in the order it see